plugins/inputs/opcua/opcua_util.go

```diff
@@ -31,11 +31,11 @@ func newTempDir() (string, error) {
 	return dir, err
 }
 
-func generateCert(host string, rsaBits int, certFile, keyFile string, dur time.Duration) (string, string) {
+func generateCert(host string, rsaBits int, certFile, keyFile string, dur time.Duration) (cert string, key string, err error) {
 	dir, _ := newTempDir()
 
 	if len(host) == 0 {
-		log.Fatalf("Missing required host parameter")
+		return "", "", fmt.Errorf("Missing required host parameter")
```

Comment:
Error strings should not be capitalized, I think I saw a couple of other places with caps as well.

Reference: https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
Yup thanks, fixed